### ISSUE TITLE
libWrapper warning message warns about conflict with Monk's Little Details

### ISSUE THREAD

**Issue by MorganVergara** [DFreds/dfreds-convenient-effects#117]
**Describe the bug**
Getting a console warning and warning message from libWrapper about interaction between DCE and Monk's Little Details.

**To Reproduce**
Happens after refresh, when assigning a new condition via UI.

**Expected behavior**
No error message should appear.

**Screenshots**
UIMessage:
![image](https://user-images.githubusercontent.com/30846057/151244167-38d3e197-84a5-46e6-940b-d955747eaa66.png)
Console:
![image](https://user-images.githubusercontent.com/30846057/151244210-6ffd4a40-78c8-4c8c-8324-c691117d9f1b.png)

**Additional context**

**Comment by DFreds** [comment]
Weird, I'm not seeing this despite using both mods myself. Can you show me an image of the settings for both Little Details and Convenient Effects?

**Comment by MorganVergara** [comment]
Certainly. 
Here's DCE
![image](https://user-images.githubusercontent.com/30846057/151254624-38b73493-bca9-457c-b943-65f714b77b03.png)

Little Details (not so little... five screenshots!)
![image](https://user-images.githubusercontent.com/30846057/151254692-67af910b-25a1-4a3a-a9f7-571c634c56fa.png)
![image](https://user-images.githubusercontent.com/30846057/151254707-3b92c5a8-4e50-4eed-a0b0-d159230ba8a3.png)
![image](https://user-images.githubusercontent.com/30846057/151254721-2652dfb8-d494-43e7-a7f1-b42c3eb16766.png)
![image](https://user-images.githubusercontent.com/30846057/151254733-a3fc8ca2-7bad-4af6-a029-7c0f20161fd1.png)
![image](https://user-images.githubusercontent.com/30846057/151254755-7def9476-b9db-48ec-a606-f4187d073190.png)

Hope that helps!

Morgan

**Comment by MorganVergara** [comment]
Ok, a bit more work and I think I figured out the interaction. In addition to DCE and Monk's Little Details, I also have Combat Utility Belt installed.  CUB has all the system conditions included by default.

Right after I reloaded, I right clicked on the test icon, then clicked a condition from the token condition UI. That's when I saw the error. I went into Condition Lab, removed all the default system conditions, and let DCE add them back to the token UI and the problem disappeared.
